index.js

```diff
@@ -1,12 +1,18 @@
-
-
 document.querySelector('button').addEventListener('click', async () => {
+  const button = document.getElementById('boom');
+  button.style.height = '150px';
+  button.style.width = '200px';
+
 	await Tone.start()
-    console.log('audio is ready')
-    //create a synth and connect it to the master output (your speakers)
-    var synth = new Tone.Synth().toMaster()
+  //create a synth and connect it to the master output (your speakers)
+  var synth = new Tone.Synth().toMaster()
 
-    //play a middle 'C' for the duration of an 8th note
-    synth.triggerAttackRelease('C4', '8n')
+  //play a middle 'C' for the duration of an 8th note
+  synth.triggerAttackRelease('C4', '8n')
 })
```

Comment:
- I would suggest that you define the callback separately and then assign that to the button event because in this way the code is more manageable and readable.
- Avoid using var when possible.
- In your callback you are receiving the target DOM element in the passed event, so you can avoid the getElementById with this syntax: **const playTone = ({target: button}) => {...}** This way you will have available the (target) button inside your function. Check this out: [StackOverflow: What exactly is the parameter e (event) and why pass it to JavaScript functions?](https://stackoverflow.com/a/35936912)

```suggestion
const playTone = ({target: button}) => {
  button.style.height = '150px';
  button.style.width = '200px';
  
 //create a synth and connect it to the master output (your speakers)
  const synth = new Tone.Synth().toMaster()  
  
 //play a middle 'C' for the duration of an 8th note
  synth.triggerAttackRelease('C4', '8n')  
}

document.querySelector('button').addEventListener('click', playTone)
```